src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,48 @@
+package homework_2.pyramid_printer;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+@Getter
+@Setter
```

Comment:
Doesn't look good. These are mostly for models

Reply:
Changed for @Data.